.vscodeignore

```diff
@@ -8,3 +8,8 @@ vsc-extension-quickstart.md
 **/.eslintrc.json
 **/*.map
 **/*.ts
+CODEOWNERS
+code-fixtires/**
+coverage/**
+github-actions/**
```

Comment:
If we're putting `github-actions` here, do we also want to put `.git2gus` and `.github`?

Reply:
Yes, they can go away as well.